tasks/run-collectors/run-collectors.yaml

```diff
@@ -62,34 +62,32 @@ spec:
 
         git clone "${COLLECTORS_REPOSITORY}" collectors
         pushd collectors
+        
+        COLLECTORS_RESOURCE_TYPE=$(echo "$COLLECTORS_RESOURCE_TYPE" | tr '[:upper:]' '[:lower:]')
```

Comment:
Why do this? I only see this used in lines 69 and 71, compared against a hardcoded string. Why not just change the hardcoded string to fit the capitalization of the task param?

Reply:
I did it this way because I wanted it to be flexible in case we change how we pass it from the service. Right now that is hardcoded in the service and we pass either `releaseplan` or `releaseplanadmission`. If as part of the other PR review or as part of a change in the future we change that, we would have to modify this piece of code.

Happy to remove this and just expect to get one of the hardcoded values. Do you want me to do so?